findbugs/build.gradle

```diff
@@ -189,6 +196,7 @@ task smokeTest {
       sourcePath(path:'src/main/java:src/gui:src/test/java:src/tools')
       'class'(location:project.tasks['compileJava'].destinationDir)
       auxClasspath(path:project.fileTree(dir:'lib', include:'**/*.jar').asPath)
+      configurations.compile.each { File file -> auxClasspath(path:file.path) }
```

Comment:
#109 deleted jar file from `findbugs/lib` directory, so we need to add `configurations.compile` explicitly.